editor/src/core/property-controls/property-controls-local.spec.tsx

```diff
@@ -1133,9 +1219,12 @@ describe('Lifecycle management of registering components', () => {
       expect(renderResult.getEditorState().editor.propertyControlsInfo['/src/card']).toBeUndefined()
     })
     it('Renaming a regular file to a component descriptor name adds the property controls', async () => {
-      const descriptorFileContent1 = `const Components = {
+      const descriptorFileContent1 = `import { Card } from '../src/card'
+      
+      const Components = {
       '/src/card': {
         Card: {
+          component: Card,
           supportsChildren: false,
           properties: {
```

Comment:
I would add a new test case where the validation fails and the component is not registered.